public/css/agileui.less

```diff
@@ -216,7 +211,7 @@ footer.atk-footer {
     width: 40%;
     z-index: @rightPanelIndex;
     &.atk-visible {
-        box-shadow: 0px 0 5px fade(@black, 20%);
+        box-shadow: 0 0 5px fade(@black, 20%);
```

Comment:
this css value is not passing styleling, introduced in https://github.com/atk4/ui/commit/03f0ab2027e8bcd6c2a2b27d95c058e7bf082566

@ibelar any idea how to fix it?

here is online repro: https://lesscss.org/less-preview/#eyJjb2RlIjoiI2xpYigpIHtcbiAgICAuY29sb3JzKCkge1xuICAgICAgQHByaW1hcnk6IGJsdWU7XG4gICAgICBAc2Vjb25kYXJ5OiBncmVlbjtcbiAgICB9XG4gICAgLnJ1bGVzKEBzaXplKSB7XG4gICAgICBib3JkZXI6IEBzaXplIHNvbGlkIHdoaXRlO1xuICAgIH1cbiAgfVxuICBcbiAgLmJveCB3aGVuICgjbGliLmNvbG9yc1tAcHJpbWFyeV0gPSBibHVlKSB7XG4gICAgd2lkdGg6IDEwMHB4O1xuICAgIGhlaWdodDogKCR3aWR0aCAvIDIpO1xuICB9XG4gIFxuICAuYmFyOmV4dGVuZCguYm94KSB7XG4gICAgQG1lZGlhIChtaW4td2lkdGg6IDYwMHB4KSB7XG4gICAgICB3aWR0aDogMjAwcHg7XG4gICAgICAjbGliLnJ1bGVzKDFweCk7XG4gICAgfVxuICB9IiwiYWN0aXZlVmVyc2lvbiI6IjQueCJ9

Reply:
did you try like this:         
box-shadow: 0px 0px 5px fade(@black, 20%);

Reply:
LESS can render it, but what should it do?

docs https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow

Reply:
https://lesscss.org/less-preview/#eyJjb2RlIjoiQGJsYWNrOiAjMDAwO1xuXG5saWIge1xuICAgIGJveC1zaGFkb3c6IDAgMCA1cHggZmFkZShAYmxhY2ssIDIwJSk7XG59IiwiYWN0aXZlVmVyc2lvbiI6IjQuMS4zIn0= - it seems to be some LESS macro but Stylelint does not like it...